app/models/experimentation/experiment.rb

```diff
@@ -21,6 +21,14 @@ class Experiment < ActiveRecord::Base
       stale_patients: "stale_patients"
     }, _prefix: true
 
+    def self.candidate_patients
```

Comment:
This could also live in Patient, maybe, but its pretty specific to experiments

Reply:
Yeah, I'll agree with that.